pkg/services/ngalert/store/database.go

```diff
@@ -55,4 +55,8 @@ func ProvideDBStore(
 		AccessControl:    access,
 		DashboardService: dashboards,
 	}
```

Comment:
Can you please change logger to `ngalert.dbstore` so it will match our other loggers?